README.md

```diff
@@ -9,14 +9,14 @@ The Open Data Hub community abides by the [Open Data Hub Code of Conduct](https:
 
 Here is an excerpt from the code of conduct:
 
-*In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.*
+> *In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.*
 
 - **Slack** - Join us on Slack for discussions and questions at [odh-io.slack.com](https://odh-io.slack.com). [Invite](https://join.slack.com/t/odh-io/shared_invite/zt-13hp18gxj-Yb34PfQyP9GDmKMU7AkVYw)
```

Comment:
I also tried to track down this link/use this invite, and got a 404. We might just need a Slack Workspace admin to update the above link with a newer token?